addons/web/static/src/js/components/filter_generator_menu.js

```diff
@@ -9,6 +9,39 @@ odoo.define('web.FilterGeneratorMenu', function (require) {
 
     const { useState } = owl.hooks;
 
+    /**
+     * Filter generator menu
+     *
+     * Component which purpose is to generate new filters from a set of inputs.
+     * It is made of one or several `condition` objects that will be used to generate
+     * filters which will be added in the filter menu. Each condition is composed
+     * of 2, 3 or 4 inputs:
+     *
+     * 1. FIELD (select): the field used as the chain in the domain condition;
+     *
+     * 2. OPERATOR (select): the operator of the domain condition, linking the field
```

Comment:
not exactly true -> is between. Rewrite that part?

Reply:
Ok